pom.xml

```diff
@@ -1970,6 +1970,9 @@
                     <artifactId>spotless-maven-plugin</artifactId>
                     <version>${maven.plugin.spotless.version}</version>
                     <configuration>
+                        <upToDateChecking>
+                            <enabled>true</enabled>
+                        </upToDateChecking>
```

Comment:
what does "upToDate" mean?

Reply:
This is a new feature supported in version 2.18.0 and indicate incremental checking, see : https://github.com/diffplug/spotless/pull/935

Reply:
Thanks